atomic_defi_design/qml/Exchange/Trade/SimpleView/List.qml

```diff
@@ -0,0 +1,299 @@
+import "Main.js" as Main
+
+DexListView {
+    id: order_list_view
+    anchors.fill: parent
+    model: API.app.orders_mdl.orders_proxy_mdl
+    clip: true
+    currentIndex: -1
+    delegate: ClipRRect {
+        property var details: model
+        readonly property bool is_placed_order: !details ? false :
+                               details.order_id !== ''
+
+        property bool expanded: order_list_view.currentIndex === index
+        width: order_list_view.width
+        height: expanded? colum_order.height+10 : 35
+        radius: 1
+        Rectangle {
+            anchors.fill: parent
+            color: order_mouse_area.containsMouse? theme.surfaceColor : 'transparent'
+            border.color: theme.surfaceColor
+            border.width: expanded? 1 : 0
+        }
+        DexMouseArea {
+            id: order_mouse_area
+            anchors.fill: parent
+            hoverEnabled: true
+            onClicked: {
+                if(order_list_view.currentIndex === index) {
+                    order_list_view.currentIndex = -1
+                }else {
+                    order_list_view.currentIndex = index
+                }
+            }
+        }
+        Column {
+            id: colum_order
+            width: parent.width
+            spacing: 5
+            RowLayout {
+                width: parent.width
+                height: 30
+                spacing: 5
+                Item {
+                    Layout.preferredWidth: 30 
+                    height: 30
+                    BusyIndicator {
+                        width: 25
+                        height: width
+                        anchors.centerIn: parent
+                        running: !isSwapDone(details.order_status) && Qt.platform.os != "osx"
+                        DefaultText {
+                            anchors.centerIn: parent
+                            font.pixelSize: 9
+                            color: !details ? "white" : getStatusColor(details.order_status)
+                            text_value: !details ? "" :
+                                        visible ? getStatusStep(details.order_status) : ''
+                        }
+                    }
+                }
+                DefaultImage {
+                    id: base_icon
+                    source: General.coinIcon(!details ? atomic_app_primary_coin :
+                                                        details.base_coin?? atomic_app_primary_coin)
+                    Layout.preferredWidth: Style.textSize1
+                    Layout.preferredHeight: Style.textSize1
+                    Layout.alignment: Qt.AlignVCenter
+                }
+                DefaultText {
+                    id: base_amount
+                    text_value: !details ? "" :
+                                General.formatCrypto("", details.base_amount, details.base_coin)
+                    //details.base_amount_current_currency, API.app.settings_pg.current_currency
```

Comment:
If you commented some code and want to keep it, you may want to add a statement which says why you commented it. If you have nothing to say about it, it is probably useless to keep it.